index.html

```diff
@@ -1,232 +1,326 @@
-                                </a>
-                            </li>
-                            <li>
-                                <a href="https://www.linkedin.com/in/oscah-deus-757833210/">
-                                    <img src="./images/Linkedin.png" alt="">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="#">
-                                    <img src="./images/Vector1.png" alt="">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="https://github.com/Oscah01/">
-                                    <img src="./images/Vector2.png" alt="p">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="#">
-                                    <img src="./images/Vector3.png" alt="p">
-                                </a>
-                            </li>
-                        </ul>
-                    </div>
-                </section>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Oscar's Deus</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lobster&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <nav class="top-sec">
+        <a href="#"><h1 class="nav-branding">Oscar Deus</h1></a>
+        <ul class="nav-menu">
+          <li class="nav-item nav-link"><a href="#">Portfolio</a></li>
+          <li class="nav-item nav-link"><a href="#">About</a></li>
+          <li class="nav-item nav-link"><a href="#">Contact</a></li>
+        </ul>
+        <div class="hambuger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+
+    <section class="homepage">
+      <div class="headtitle-name headtitle">
+        <h1 class="primary-text">I'm Oscar<br />Glad to see you!</h1>
+        <p class="phar">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
```

Comment:
- [ ] Great job here keep it up but I think you just need to adjust the background here to match the Figma design🔥🔥🔥🔥

- [ ] The font-weight here for lets connect is not the same please Fix it ✅✅✅

- Your design 
![image](https://user-images.githubusercontent.com/58553711/179329759-a787e21d-d5d8-4044-a91d-5f6a0798b275.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179329803-124fb07c-f00d-41ed-9354-daede15d1389.png)